core/src/components/chip/chip.scss

```diff
@@ -120,6 +120,21 @@
   @include margin(-4px, -8px, -4px, 8px);
 }
 
+// Chip Shapes
+// ---------------------------------------------
+
+:host(.chip-soft) {
+  --border-radius: 8px;
+}
+
+:host(.chip-round) {
+  --border-radius: 999px;
+}
+
+:host(.chip-rectangular) {
+  --border-radius: 0px;
```

Comment:
I hardcoded these values as I assumed we didn't want to use the design tokens in `ios` / `md` but I can update them if we do.

Reply:
I'm okay with this. We don't seem to use design tokens when it comes to border radius in other components for `ios` or `md`. So this approach follows the current structure.